frontend/include/chpl/framework/Context-detail.h

```diff
@@ -238,12 +238,21 @@ class QueryMapResultBase {
   mutable RevisionNumber lastChanged;
 
   mutable QueryDependencyVec dependencies;
+
+  // Whether or not errors from this query result have been shown to the
+  // user (they may not have been if some query checked for errors).
+  mutable bool emittedErrors;
+  // Whether or not this query result is a point at which error messages
+  // began being collected (and thus not reported to the user).
+  mutable bool errorCollectionRoot;
```

Comment:
Let's set these with `= false` here and also give defaults for `lastChecked = -1` and `lastChanged = -1` above. (These are nice because they make sure all initializers set them).